<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>About</class>
 <widget class="QWidget" name="About">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>395</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-image: url(:/A31/A31/35_all/67.bmp);</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 20pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>关于</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/A31/A31/35_all/67.bmp);
color: rgb(255, 255, 255);
font: 75 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_SN">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>397</y>
     <width>114</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_43">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>440</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>版  本：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_53">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>350</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>型  号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_model">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>114</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>A31</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_41">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>397</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>序列号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_82">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>2</y>
     <width>771</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>上海耀华称重系统有限公司

地  址： 上海市浦东新区上南路4059号

电  话： 021-67282812

传  真： 021-67282830

邮  箱：  xs@yaohua.com.cn

邮  编：  201112 

网  址：  www.yaohua.com.cn </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_version">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>440</y>
     <width>128</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>V1.0001</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
